cmd/fyne/commands/release.go

```diff
@@ -36,9 +39,10 @@ func (r *releaser) AddFlags() {
 	flag.StringVar(&r.appVersion, "appVersion", "", "Version number in the form x, x.y or x.y.z semantic version")
 	flag.IntVar(&r.appBuild, "appBuild", 0, "Build number, should be greater than 0 and incremented for each build")
 	flag.StringVar(&r.keyStore, "keyStore", "", "Android: location of .keystore file containing signing information")
-	flag.StringVar(&r.certificate, "certificate", "Apple Distribution", "iOS/macOS/Windows: name of the certificate to sign the build")
+	flag.StringVar(&r.certificate, "certificate", "", "iOS/macOS/Windows: name of the certificate to signAndroid the build")
```

Comment:
nit: find/replace error
```suggestion
	flag.StringVar(&r.certificate, "certificate", "", "iOS/macOS/Windows: name of the certificate to sign the build")
```

Reply:
Could certificate flag be used for android if the keystore contains multiple?

Reply:
The keystore command for Android will confirm which alias to use.
This may be adapted > 1.4 for fyne-cross 1.0.0, but I left it out for now as not needed.

Reply:
Good catch, turns out ther were two of them!